src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -313,16 +313,22 @@ typedef enum {
 } uct_ib_mlx5_mmio_mode_t;
 
 
+typedef enum {
+    UCT_IB_MLX5_CQE_ZIPPING_RX,
+    UCT_IB_MLX5_CQE_ZIPPING_TX,
+    UCT_IB_MLX5_CQE_ZIPPING_LAST
+} uct_ib_mlx5_cqe_zipping_mode_t;
```

Comment:
Maybe we can remove it at all and use `uct_ib_dir_t` instead?
Then `uct_ib_mlx5_parse_cqe_zipping` will be simpler